libraries/AC_Avoidance/AP_OABendyRuler.cpp

```diff
@@ -19,12 +19,52 @@
 #include <AP_AHRS/AP_AHRS.h>
 #include <AP_Logger/AP_Logger.h>
 
+const float OA_BENDYRULER_LOOKAHEAD_DEFAULT = 15.0f;
+const float OA_BENDYRULER_RATIO_DEFAULT = 1.5f;
+const int16_t OA_BENDYRULER_ANGLE_DEFAULT = 75;
+
 const int16_t OA_BENDYRULER_BEARING_INC = 5;            // check every 5 degrees around vehicle
 const float OA_BENDYRULER_LOOKAHEAD_STEP2_RATIO = 1.0f; // step2's lookahead length as a ratio of step1's lookahead length
 const float OA_BENDYRULER_LOOKAHEAD_STEP2_MIN = 2.0f;   // step2 checks at least this many meters past step1's location
 const float OA_BENDYRULER_LOOKAHEAD_PAST_DEST = 2.0f;   // lookahead length will be at least this many meters past the destination
 const float OA_BENDYRULER_LOW_SPEED_SQUARED = (0.2f * 0.2f);    // when ground course is below this speed squared, vehicle's heading will be used
 
+const AP_Param::GroupInfo AP_OABendyRuler::var_info[] = {
+
+    // @Param: LOOKAHEAD
+    // @DisplayName: Object Avoidance look ahead distance maximum
+    // @Description: Object Avoidance will look this many meters ahead of vehicle
+    // @Units: m
+    // @Range: 1 100
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("LOOKAHEAD", 1, AP_OABendyRuler, _lookahead, OA_BENDYRULER_LOOKAHEAD_DEFAULT),
+
+    // @Param: BENDY_RATIO
+    // @DisplayName: Margin ratio for BendyRuler to change bearing significantly 
+    // @Description:  BendyRuler will avoid changing course unless ratio of previous margin to present calculated margin is atleast this much.
```

Comment:
If possible I try to not use different words for the same thing to reduce user confusion.  So for example, "yaw", "bearing", "heading" and "course" are all used interchangeably.  Let's stick as much as possible to just one word.  In general we use "Yaw" but it's possible that BendyRuler using another word.  I don't mind too much which one we pick but let's try and pick just one word for the UI part at least.